Split/Network/HttpClient/HttpClient.swift

```diff
@@ -249,4 +246,9 @@ extension HttpRequestManager: URLSessionDataDelegate {
             request.appendData(data)
         }
     }
+
+    public func urlSession(_ session: URLSession, dataTask: URLSessionDataTask,
```

Comment:
is it okey this function with no statements?

Reply:
mmm, not sure what are you pointing at... but that _public_ clause should not be there. Removing...